vsintegration/src/FSharp.Editor/Diagnostics/UnusedOpensDiagnosticAnalyzer.fs

```diff
@@ -26,7 +26,7 @@ type internal UnusedOpensDiagnosticAnalyzer [<ImportingConstructor>] () =
 
             let! _, checkResults =
                 document.GetFSharpParseAndCheckResultsAsync(nameof (UnusedOpensDiagnosticAnalyzer))
-                |> liftAsync
+                |> CancellableTask.start CancellationToken.None
```

Comment:
Why None?

Reply:
I have missed this one, should've been lifted from async's one.